app/assets/javascripts/angular/directives/graph_chart.js

```diff
@@ -277,6 +278,19 @@ angular.module("Prometheus.directives").directive('graphChart', ["$location", "W
         rsGraph.series.legend = legend;
         rsGraph.render();
 
+        var drag = new Rickshaw.Graph.DragZoom({
+          graph: rsGraph,
+          opacity: 0.5,
+          fill: 'steelblue',
+          minimumSelection: 25,
```

Comment:
Woud it be possible to use seconds as the minimum selection unit?

It's currently possible to zoom in into a total range <1s with values in ms as unit in the x axis. I guess it doesn't make sense to have a total range smaller than say 5s. Using time as absolute minimum instead of pixel width which is relative to the current state ensures zooming determines.